### PR TITLE
Feature/ping pong

### PR DESCRIPTION
**Pull request by dchilot**

### CODE REVIEW

File: include/orwell/callbacks/ProcessPing.hpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{
+
+namespace callbacks
+{
+
+class ProcessPing : public InterfaceProcess
+{
+public:
+	ProcessPing(
+			std::shared_ptr< com::Sender > ioPublisher,
```
Comment (miludmann) [inline include/orwell/callbacks/ProcessPing.hpp]: tabulation looks weird in this file...

File: include/orwell/callbacks/ProcessPong.hpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+
+namespace callbacks
+{
+
+class ProcessPong : public InterfaceProcess
+{
+public:
+	ProcessPong(
+			std::shared_ptr< com::Sender > ioPublisher,
```
Comment (miludmann) [inline include/orwell/callbacks/ProcessPong.hpp]: tabulation looks weird in this file too...

File: src/orwell/callbacks/ProcessHello.cpp [hunk 3]
```diff
@@ -86,12 +86,20 @@ void ProcessHello::execute()
 				m_publisher->send(aReply);
 				aFailure = false;
 
+				if (aHelloMessage.has_ready())
+				{
+					ORWELL_LOG_INFO("Hello ready ? " << aHelloMessage.ready());
```
Comment (miludmann) [inline src/orwell/callbacks/ProcessHello.cpp]: Whitespace typo: "Hello ready?"

Comment (dchilot) [reply src/orwell/callbacks/ProcessHello.cpp]: Well, it is not a typo. I really prefer to have a space there.

File: include/orwell/callbacks/ProcessPong.hpp (hunk 2 above)
Comment (dchilot) [reply include/orwell/callbacks/ProcessPong.hpp]: My current configuration puts two tabulations to indent this. I am not sure if it is bad or good.

**Pull request merged**